components/helpers/interfaces.tsx

```diff
@@ -39,6 +39,7 @@ export interface Picture {
   picture_path: string
 }
 export interface Collection {
+  id?: number
```

Comment:
isn't there UUID or somethin for id in typescript?

Reply:
there's nothing built in, in typescript